confd/templates/server42/server42-control.application.properties.tmpl

```diff
@@ -41,3 +41,9 @@ spring.kafka.consumer.properties.spring.json.default.value.type=org.openkilda.se
 spring.kafka.consumer.properties.spring.json.use.type.headers=false
 spring.kafka.consumer.properties.spring.json.remove.type.headers=true
 spring.kafka.listener.missing-topics-fatal=false
+
+# Kafka versioning (zero_downtime)
+spring.kafka.producer.properties.interceptor.classes=org.openkilda.bluegreen.kafka.interceptors.VersioningProducerInterceptor
+spring.kafka.producer.properties.kafka.producer.messaging.zookeeper.connecting.string.property={{ getv "/kilda_zookeeper_hosts"}}/{{ getv "/kilda_zookeeper_state_root" }}
```

Comment:
Extreme amount of kafka, producer and properties fords in one... name / definition :)

Reply:
agree. looks ugly. Let me explain. 
it consists of two parts:
1. `spring.kafka.producer.properties`
2. `kafka.producer.messaging.zookeeper.connecting.string.property`

Second one is mine. First one is spring's. I'm glad to use some simple name, but there is a problem. These props are shared.
from docs ` be aware that it will be sharing config namespace with other interceptors and serializers`
So I need a unique name. 
In other parts of code it looks okay because I used only second part there, but in server42 I had to use spring configs